idea/Hublsoft.Task.Web/Data/ApplicationDbContext.cs

```diff
@@ -12,5 +13,7 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
             : base(options)
         {
         }
+
+        public DbSet<LoginMonitorModel> UserLoginMonitors { get; set; }
```

Comment:
trying to get this up and running and getting connection failures. There's nothing in the readme about how to set up this application - do i just point the `Default` connection string at my local sql server?

Reply:
![image](https://user-images.githubusercontent.com/910448/92402489-8d02a200-f127-11ea-856c-583a825a4e12.png)

Reply:
Apologies i will add the EF migration notes to the read me file, which should create the database and schema for you and allow the application to run.

Reply:
ok this is working now. Tweaked the readme to clarify.